components/PicklistBase/PicklistBase.vue

```diff
@@ -36,7 +36,8 @@
                 variant="primary"
                 v-on:click="handleUnitsButton()"
               >
-                {{ units }}
+                <span v-if="allPicked">🚫 {{ units }}</span>
```

Comment:
We need something similar above for the "All" button when `units` are not being used.  To see this go to the `PicklistBase` example page and click the "useUnits" `prop` control to switch to not using `units`. In that case the button just contains "All".  It should have the icon same as when we are using `units`.

The test cases will need to be updated to also include tests for the context sensitive icon when `units` are not being used.  They will parallel the ones for when `units` are being used.